cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureSetup.java

```diff
@@ -158,6 +160,20 @@ public void validateFileSystem(CloudCredential credential, SpiFileSystem spiFile
         }
     }
 
+    @Override
+    public void validateParameters(AuthenticatedContext ac, Map<String, String> parameters) throws Exception {
+        AzureClient client = ac.getParameter(AzureClient.class);
+        String resourceGroupName = parameters.get(AzureResourceConnector.RESOURCE_GROUP_NAME);
+        if (StringUtils.isEmpty(resourceGroupName)) {
+            return;
+        }
+        ResourceGroup resourceGroup = client.getResourceGroup(resourceGroupName);
+        if (resourceGroup != null) {
+            throw new BadRequestException("ResorceGroupName is exists: " + resourceGroup.name());
```

Comment:
Suggestion: "Resourcegroup name already exists: "

Reply:
or 'Resource group with {%name} already exists on provider side, please choose a different one.'

Reply:
@pdarvasi what do you think ?